db/resourceDBInterface.go

```diff
@@ -0,0 +1,121 @@
+package db
+
+import (
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"os"
+)
+
+type CResource struct {
+	ID    primitive.ObjectID `bson:"_id,omitempty" json:"_id,omitempty"`
+	Name  string             `bson:"name" json:"name"`
+	Type  string             `bson:"type" json:"type"`
+	Users []User             `bson:"users" json:"users"`
+}
+
+var collectionName = "cResources"
+
+func GetAllResources() ([]CResource, error) {
```

Comment:
For the sake of consistency the file should be c_resourceDBInterface.go (the underscore because that's how we did it in /handlers though now I'm wondering whether that might've been a bad choice since we use camelcase everywhere else) 

and functions should be
GetAllCResources
GetCResourceById etc.

Same for variables.

Reply:
Resolved in new commit. Dont really mind the underscore.